### PR TITLE
[UP][m] Updating the scripts with v2 source of world bank

### PR DESCRIPTION
**Pull request by gradedSystem**
## Changes made

- Updated the scripts to update the `archive` and `data` folders 
- Updated requirements.txt
- Small README.md update

cc @anuveyatsu

### CODE REVIEW

**Review by Mikanebu — CHANGES_REQUESTED** [review]
Hey @gradedSystem ! Nice work, but lets remove this README in `scripts` and merge it into README in root directory, all the contents move to `Preparation` section as in https://github.com/datasets/currency-codes. Also include how to run `make` file as in your github actions.  Lets make a same structure. Also please remove some unnecessary headings such as `Usage of process.py` and `Tags and committing changes` and make it simple and easy to read.

**Comment by gradedSystem** [comment]
@Mikanebu done

File: README.md [hunk 1]
```diff
@@ -7,9 +7,40 @@ The repository of the data package of the GINI Index.
...
+
+## Contributing
+
+This dataset will eventually need an update, specially because there are many
+pieces missing.
+In order to contribute, please fork this repository and do the changes you must,
+but please follow [OKFN's curator guide](http://data.okfn.org/doc/core-data-curators)
+to keep things as organized as possible.
+
+## Tags and committing changes
```
Comment (Mikanebu) [inline README.md]: Remove this pls

File: README.md [hunk 2]
```diff
@@ -7,9 +7,40 @@ The repository of the data package of the GINI Index.
...
+
 
 ## License
 
 All data is licensed under the Open Data Commons Public Domain Dedication and License. All code is licensed under the MIT/BSD license.
 
 Note that while no credit is formally required a link back or credit to Rufus Pollock and the Open Knowledge Foundation is much appreciated.
+
+
+## Contributing
```
Comment (Mikanebu) [inline README.md]: Also this one, since that url is not valid anymore

File: README.md [hunk 3]
```diff
@@ -7,9 +7,40 @@ The repository of the data package of the GINI Index.
 The data that is contained in the `gini-index.csv` file, under `/data` was
 retrieved from [the World Bank](http://data.worldbank.org/indicator/SI.POV.GINI).
 
+## Requirements
```
Comment (Mikanebu) [inline README.md]: This one should be `Preparation` section and add some sentence on how to run scripts

File: README.md [hunk 4]
```diff
@@ -7,9 +7,40 @@ The repository of the data package of the GINI Index.
...
+
+Run the requirements through `pip` in order to install all required packages to 
+run the script.
+
+`pip install -r scripts/requirements.txt`
+
 
 ## License
```
Comment (Mikanebu) [inline README.md]: Before the license, add `Automation` section as in https://github.com/datasets/gdp

**Review by Mikanebu — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -7,9 +7,20 @@ The repository of the data package of the GINI Index.
 The data that is contained in the `gini-index.csv` file, under `/data` was
 retrieved from [the World Bank](http://data.worldbank.org/indicator/SI.POV.GINI).
 
+## Preparation
 
-## License
+Run the requirements through `pip` in order to install all required packages to 
+run the script.
+
+`pip install -r scripts/requirements.txt`
```
Comment (Mikanebu) [inline README.md:14]: after this add how to run script for example:
```
Process is recorded and automated in python script:
scripts/process.py
```

**Review by Mikanebu — CHANGES_REQUESTED** [review]
Please, add how to run script in `Preparation` section

File: README.md (hunk 5 above)
Comment (gradedSystem) [reply README.md:14]: done @Mikanebu

**Review by Mikanebu — CHANGES_REQUESTED** [review]
In your actions, it is `make` to run script?

**Comment by gradedSystem** [comment]
@Mikanebu Yes it is

File: README.md (hunk 5 above)
Comment (Mikanebu) [reply README.md:14]: update to `make` pls

**Comment by Mikanebu** [comment]
@gradedSystem Pls, update then accordingly.

**Review by Mikanebu — APPROVED** [review]
LGTM!

**Pull request merged**